### PR TITLE
bugfix(pipeline): Add commit to main to update CHANGELOG

### PR DESCRIPTION
**Pull request by pramirezma**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My changes generate no new warnings or errors

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature Implementation
- [x] :bug: Bugfix
- [ ] :art: Code Formatting (formatting, renaming)
- [ ] :recycle: Refactoring
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :hammer: Automated testing
- [ ] :zap:  Dependency update
- [ ] :wrench: Add or update configuration files

## What is the current module?

-

## What is the new behavior?

- This add a step in the pipeline that updates the changelog every time there is a new release

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by Lmam22 — APPROVED** [review]

File: .github/workflows/merge-master.yml [hunk 1]
```diff
@@ -14,10 +16,13 @@ jobs:
...
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 0
+          fetch-depth: 2
       - name: Get commit message
         id: commit
-        run: echo "COMMIT_MSG=$(git log --format=%B -n 1 ${{ github.event.after }})" >> $GITHUB_OUTPUT
+        run: |
+          COMMIT_MSG=$(git log --format=%B -n 1 HEAD| tail -n 1)
+          echo "Merge commit message pepito: $COMMIT_MSG"
```
Comment (jose-sal-y-rosas) [inline .github/workflows/merge-master.yml:24]: What is the meaning of 'pepito'? is a magic word?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please request changes

File: .github/workflows/merge-master.yml (hunk 1 above)
Comment (pramirezma) [reply .github/workflows/merge-master.yml:24]: nothing, shouldn't be there, i can't even image how it did it get there hahaha

**Review by pamelaluquemck — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**